src/components/calculator.js

```diff
@@ -0,0 +1,28 @@
+function Calculator() {
+  return (
+    <div className="calculator-grid">
+      <div className="result">0</div>
+      <button type="button">AC</button>
+      <button type="button">+/-</button>
+      <button type="button">%</button>
+      <button type="button" className="orange">/</button>
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="orange">x</button>
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button" className="orange">-</button>
+      <button type="button">1</button>
+      <button type="button">2</button>
+      <button type="button">3</button>
+      <button type="button" className="orange">+</button>
+      <button type="button" className="span">0</button>
+      <button type="button">.</button>
+      <button type="button" className="orange">=</button>
+    </div>
+  );
+}
+
+export default Calculator;
```

Comment:
- [ ] It is required that this Calculator component holds at least one react component.
![image](https://user-images.githubusercontent.com/86200356/233972574-a45caabf-b13b-4d88-b1f6-8f3343b9fd41.png)

I would implore you to extract the html tags here into another component you will create and import it into this component.

- [OPTIONAL] You can have your data in an array and map over the array rather than repeatedly writing the tags.

Reply:
@Olamarx thanks for the review, but I am not understanding.
Is this not a react component?

Reply:
@Olamarx could you give me another example of a component I could add that does not involve me moving my divs to another file kindly?

Reply:
Kindly note that every component is going to show on the UI so the easiest of it is to just move the content of Calculator.js to another component and import it here just as you did import Calculator.js inside App.js, then you are good to go.

Reply:
@Olamarx  I think I have done it. Could you kindly pull and see if it's okay before I submit another review request?

Reply:
I think you have too, I can see the Display component but I can't pull it at the moment, kindly remember to delete the demo file also.

Reply:
I have removed the demo file. Let me then request review and see